### PR TITLE
Delete duplicate settings

### PR DESCRIPTION
**Pull request by kuroponzu**
_Before contributing, please read our [contributing guidelines](https://github.com/newrelic/newrelic-ruby-agent/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/newrelic/.github/blob/master/CODE_OF_CONDUCT.md)._

# Overview
Delete duplicate settings in newrelic.yml

# Related Github Issue
https://github.com/newrelic/newrelic-ruby-agent/issues/849

# Testing
The agent includes a suite of unit and functional tests which should be used to
verify your changes don't break existing functionality. These tests will run with 
Github Actions when a pull request is made. More details on running the tests locally can be found 
[here for our unit tests](https://github.com/newrelic/newrelic-ruby-agent/blob/main/test/README.md), 
and [here for our functional tests](https://github.com/newrelic/newrelic-ruby-agent/blob/main/test/multiverse/README.md).
For most contributions it is strongly recommended to add additional tests which
exercise your changes. 

# Reviewer Checklist
- [ ] Perform code review
- [ ] Add performance label
- [ ] Perform appropriate level of performance testing
- [ ] Confirm all checks passed
- [ ] Add version label prior to acceptance

### CODE REVIEW

File: newrelic.yml [hunk 1]
```diff
@@ -32,22 +32,6 @@ common: &default_settings
...
-# Defines the maximum number of request events reported from a single harvest.
-# analytics_events.max_samples_stored: 1200
-
-# Defines the maximum number of request events reported from a single harvest.
-# analytics_events.max_samples_stored: 1200
-
-# Prefix of attributes to exclude from all destinations. Allows * as wildcard at
-# end.
-# attributes_exclude: []
-
```
Comment (kaylareopelle) [inline newrelic.yml:50]: I'm not seeing this setting elsewhere in this file. Do you? If not, could you add it back?

**Review by kaylareopelle — COMMENTED** [review]
@kuroponzu, thank you for picking up this ticket! It's a pleasure to e-meet you. We appreciate the time you took to improve the newrelic-ruby-agent's codebase! 

It looks like there was one non-duplicated setting removed. Otherwise, this looks great to me! Once it is added back, I'll approve.

File: newrelic.yml (hunk 1 above)
Comment (kuroponzu) [reply newrelic.yml]: Sorry, I erased too much.
I added it back.

https://github.com/newrelic/newrelic-ruby-agent/pull/862/commits/73774e232f2d35d26e0033240beb61a34527429a

**Review by kaylareopelle — APPROVED** [review]
Thank you for the fix, @kuroponzu! We're in the middle of a release right now so we are not currently merging anything into dev. I'll mark this as `pending-merge` so the change is included in our following release.

**Comment by kaylareopelle** [comment]
Hi @kuroponzu  - I wanted to let you know that I'm merging this into our 8.2.0 release 😄 

Thanks again for your help!

**Pull request merged**